subprojects/core/src/integTest/groovy/org/gradle/api/tasks/FailingIncrementalTasksIntegrationTest.groovy

```diff
@@ -16,9 +16,16 @@
 
 package org.gradle.api.tasks
 
+import org.gradle.api.tasks.incremental.IncrementalTaskInputs
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.test.fixtures.file.TestFile
+import org.gradle.util.TextUtil
+import org.gradle.work.InputChanges
+import org.intellij.lang.annotations.Language
+import spock.lang.Issue
 import spock.lang.Unroll
 
+@Unroll
```

Comment:
Global `@Unroll` annotations have one disadvantage: IDEA only can jump to a test from the test execution Window if the test has not been unrolled. Using a global `@Unroll` on the class means that it is impossible to jump to any test method, even if it is not parameterized.

Reply:
This really bothers me as well, but I can't jump with local `@Unroll`s either:
<img width="735" alt="Screenshot 2019-05-21 at 11 12 41" src="https://user-images.githubusercontent.com/1841944/58083507-6a0e5080-7bb9-11e9-86d6-04fe32a1a7a7.png">